crawler_mcp/crawlers/optimized/core/strategy.py

```
@@ -0,0 +1,1591 @@
+                        fb_run_cfg.delay_before_return_html = max(
+                            0.5, getattr(fb_run_cfg, "delay_before_return_html", 0.0)
+                        )
+
+                    # Single fetch
+                    first = await self.parallel_engine.crawl_batch_raw(
+                        [start_url], fb_browser, fb_run_cfg, None
+                    )
+                    if first:
+                        r = first[0]
+                        base_host = urlparse(start_url).netloc
+                        base_root = f"{urlparse(start_url).scheme}://{base_host}"
```

Ruff (0.12.2)

Local variable `base_root` is assigned to but never used

Remove assignment to unused variable `base_root`

(F841)